src/main/java/ToDo.java

```diff
@@ -0,0 +1,37 @@
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class ToDo extends Task {
+    public ToDo(String name) {
+        super(name);
+    }
+
+    public ToDo(String name, boolean isDone) {
+        super(name, isDone);
+    }
+
+    @Override
+    public String getDisplayName() {
+        return getName();
+    }
+
+    @Override
+    public String getType() {
+        return "T";
```

Comment:
Type can be declared as a field, assign respective type string representation in the constructor, and implement a generic getter method in the parent Task class to save a few more keystrokes :)